robot_name: "secondary_robot"

critical_angle: 2.09
approach_dist: 0.11
drive_back_dist: 0.04
RATE: 10
scale_factor: 10
coords_threshold: 0.01
